CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.40.6] - 2018-12-11
```

Comment:
This adds feature, I think it should be minor instead of patch

Reply:
Ah yes, correct! Changed it to 0.41.0.